### PR TITLE
Adding Windows Getting Started doc

### PR DESCRIPTION
**Pull request by danbugs**
This addresses issue #11

### CODE REVIEW

File: docs/windows-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Windows: Getting Started
+
+Currently, **runwasi** depends on POSIX environment (i.e., because it has to wire up networking and rootfs mounts). Therefore, to run it on Windows, we recommend utilizing the Windows Subsystem for Linux (WSL).
```
Comment (cpuguy83) [inline docs/windows-getting-started.md:3]: Not just POSIX, this is using Linux specific features.

Comment (danbugs) [reply docs/windows-getting-started.md]: I just changed this!

File: docs/windows-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- run `make build`,
+- run `make install`,
+- run `make test/out/img.tar`,
+- open a secondary terminal and run `containerd`, and
+- run `make load`.
+
+After this, you can execute an example, like: `ctr run --rm --runtime=io.containerd.wasmtime.v1 docker.io/library/wasmtest:latest testwasm`.
+
+> To kill the process from the example:
+>> - open a secondary terminal and run `ps aux | grep ctr`,
```
Comment (cpuguy83) [inline docs/windows-getting-started.md:25]: This should be `ctr kill -s SIGKILL testwasm`

**Comment by Mossaka** [comment]
LGTM 

nit: I would recommend setting `make build` to be `cargo build --release` by default since the released binary is much much faster than debug mode.

**Review by Mossaka — APPROVED** [review]

**Comment by cpuguy83** [comment]
@Mossaka To do this `make build TARGET=release` should do the trick.
Likewise `sudo make install TARGET=release`

File: docs/windows-getting-started.md (hunk 2 above)
Comment (danbugs) [reply docs/windows-getting-started.md:25]: Ooo~ I didn't know this was the right way to do this! Thanks (:

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**